.scalafmt.conf

```diff
@@ -1,4 +1,4 @@
-version = "2.0.0"
```

Comment:
Ah, this file can be deleted and added to `.gitignore` now. It's auto-generated by our build plugins.